openslides_backend/action/actions/user/user_mixin.py

```diff
@@ -173,3 +186,42 @@ def get_history_information(self) -> Optional[HistoryInformation]:
                     fqid_from_collection_and_id("user", instance["id"])
                 ] = instance_information
         return information
+
+
+class DuplicateCheckMixin(Action):
+    def init_duplicate_set(self, data: List[Any]) -> None:
+        self.users_in_double_lists = self.execute_presenter(
+            SearchUsers,
+            {
+                "permission_type": "organization",
+                "permission_id": 1,
+                "search": data,
+            },
+        )
+        self.used_usernames: List[str] = []
+        self.used_names_and_email: List[Any] = []
+
+    def check_username_for_duplicate(self, username: str, payload_index: int) -> bool:
+        result = (
+            self.users_in_double_lists[payload_index] or username in self.used_usernames
```

Comment:
the type of result is list, if there are elements in the self.users_in_double_lists[payload_index

Reply:
It should be bool, I add a check.